### PR TITLE
Adding solution for pip error with Anaconda

### PR DESCRIPTION
**Pull request by Carmezim**
Issue #6136.
As this is not a Windows specific problem I added the instructions on "PIp Installation Issues".

### CODE REVIEW

File: tensorflow/g3doc/get_started/os_setup.md [hunk 1]
```diff
@@ -1050,6 +1050,27 @@ Exception:
...
+`pip install --upgrade -I setuptools`
+
+2. Install TensorFlow again adding `--ignore-installed` flag:  
+`pip install --ignore-installed --upgrade <tensorflow_url>`
+
+
+
+Step #1 might already solve the problem, however if it still persists, execute step #2.
+
+This is an issue that occurs with new Ananconda installations when `pip` tries to remove `easy-install.pth`
```
Comment (Mistobaan) [inline tensorflow/g3doc/get_started/os_setup.md]: Ananconda typo

**Comment by yifeif** [comment]
@Carmezim thanks for adding this!

**Review by yifeif — APPROVED** [review]

File: tensorflow/g3doc/get_started/os_setup.md (hunk 1 above)
Comment (yifeif) [inline tensorflow/g3doc/get_started/os_setup.md]: Fixed. Thanks @Mistobaan

**Comment by Carmezim** [comment]
@yifeif glad to do it!

**Pull request merged**